### PR TITLE
📇 Updating URNs using modifiers 

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/api/v2/serializers.py [hunk 1]
```diff
@@ -655,8 +655,7 @@ def save(self):
                 mods += self.instance.update_static_groups(groups)
 
         if mods:
-            self.instance.modify(self.context["user"], *mods)
-            self.instance.refresh_from_db()
+            self.instance.modify(self.context["user"], mods)
```
Comment (rowanseymour) [inline temba/api/v2/serializers.py:658]: modify now does the refresh

File: temba/api/v2/tests.py [hunk 2]
```diff
@@ -1782,7 +1782,7 @@ def test_contacts(self, mr_mocks):
         self.assertEqual(jean.get_field_value(nickname), "Jado")
 
         # update by UUID and change all fields
-        with self.assertNumQueries(46):
+        with self.assertNumQueries(43):
```
Comment (rowanseymour) [inline temba/api/v2/tests.py:1785]: these checks definitely becoming less meaningful as mailroom does more of the work

File: temba/contacts/models.py [hunk 3]
```diff
@@ -2162,70 +2167,6 @@ def get_urn(self, schemes=None):
             # otherwise return highest priority of any scheme
             return urns[0] if urns else None
 
-    def update_urns(self, user, urns):
```
Comment (nicpottier) [inline temba/contacts/models.py:2165]: this isn't used by imports? imports aren't using modifiers yet are they?

Comment (rowanseymour) [reply temba/contacts/models.py:2165]: No they use `Contact.get_or_create_by_urns`.. which might be next on the chopping block

**Pull request merged**